packages/base/src/toolbar/usertoolbaritem.tsx

```diff
@@ -71,10 +71,21 @@ export class UsersItem extends React.Component<IProps, IState> {
     return el;
   }
 
+  private filterDuplicated(usersList: IUserData[]): IUserData[] {
+    const newList: IUserData[] = [];
+    const selected = new Set<string>();
+    for (const element of usersList) {
+      if (!selected.has(element.userData.username)) {
+        selected.add(element.userData.username);
+        newList.push(element);
+      }
+    }
+    return newList;
+  }
   render(): React.ReactNode {
     return (
       <div className="jpcad-toolbar-usertoolbar">
-        {this.state.usersList.map(item => {
+        {this.filterDuplicated(this.state.usersList).map(item => {
```

Comment:
Nice! I've seen this issue once indeed when using the Python API next to the main view